package/MDAnalysis/topology/PDBParser.py

```diff
@@ -95,6 +102,15 @@
 # Set up a logger for the PDBParser
 logger = logging.getLogger("MDAnalysis.topology.PDBParser")
 
+try:
+    import pooch
+except ImportError:
+    HAS_POOCH = False
+else:
+    HAS_POOCH = True
+
+#: Base URL for download files from the Protein Databank.
+PDB_BASE_URL = "https://files.wwpdb.org/download/"
```

Comment:
You are using this in exactlly one place in the code and the code isn't in such a way that you can optionally change this. As such, I'm not sure there's a lot of use in having this be a global variable. Can you just move this string to line 681 please?

Reply:
I now realise my comment ran counter to @orbeckst 🤦🏽 sorry - I do think this is the correct thing though (unless we offer flexibility, there's no need to make it complex).

Reply:
Don't worry @IAlibay . My thought was to have it documented where we pull from. But you make a valid point and I am totally happy to go along with your suggestion.